### DISCUSSION TITLE
2023-10-05

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#617] [daily_meeting]
### 개인 일정

- 노아 - 5시 근로

### 오늘의 할 일

- 오후 - 3시 데분당태 미팅

### 홍보 관련
- 데분당태 홍보 관련으로 왼손에게 답장받고 채팅 보내기 - 테오
- 새로운 메일 답변 쓰기 - 룩소

### 내일의 할 일

**Comment by nlom0218** [comment]
### 체크인

7점 - 갑자기 너무 추워졌어요


### 체크아웃

6점 - url search가 고통이네요.. 처음부터 생각하고 했어야 했는데ㅠㅠ

**Comment by yeopto** [comment]
### 체크인

10점 - 디지털 노마더가 된 기분입니다.

### 체크아웃

8점 - 내일은 잠실로..

**Comment by jaehee329** [comment]
### 체크인
8점 - 페인트칠하는 선릉을 갈까말까 고민중

### 체크아웃
8점 - 내일은 만나서 봐요

**Reply by MoonJeWoong** [reply]
이미 고민하는 시점에서 늦은거 아닌가요 ㅋㅋㅋ

**Comment by aak2075** [comment]
### 체크인

10점 - 출근까지 단 3초

### 체크아웃

10점 - 시간 참 빠르네요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 어제 개발할거 끝내고 자서 좋긴한데 미션좀 빨리 쳐내고 싶네요..

### 체크아웃
9점 - 미션 멸망전 들어갑니다

**Comment by woosung1223** [comment]
### 체크인
9점 - 9시 59분에 일어나서 스펙타클 합니다.

### 체크아웃
8점 - 재택 조금만 더 하면 안될까요

**Comment by woo-jk** [comment]
### 체크인
9점 - 할게 넘 많다

### 체크아웃
10점 - 서울 가야합니다
